vcpkg.json

```diff
@@ -2,14 +2,26 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "parquetsharp",
   "version-string": "undefined",
-  "builtin-baseline": "d9ccd77bb554e67137f3f754a2e2f11f4188c82c",
+  "builtin-baseline": "e08b7bd89ae162f8579df2f8d39a1ae94107c8fd",
```

Comment:
Needed to bump the baseline to pick up this fix that allows using mimalloc on Windows: https://github.com/microsoft/vcpkg/pull/46299